nominees/towi-plugin.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Towi Plugin",
+  "aliases": [
+    "towigodot"
+  ],
+  "description": "Plugin made to interact with Pixframe Studios Towi platforms which allow creators to create new Serious games and connected to Towi.com.mx",
```

Comment:
Is "Serious games" a proper noun indicating some third-party software, or does it simply mean serious games?

Reply:
Serious games is proper noun and common phenomena in the game industry referring to an applied game is a [game](https://en.wikipedia.org/wiki/Game) designed for a primary purpose other than pure [entertainment](https://en.wikipedia.org/wiki/Entertainment).[[](https://en.wikipedia.org/wiki/Serious_game#cite_note-1)

Reply:
> Is "Serious games" a proper noun indicating some third-party software, or does it simply mean serious games?

Serious games is proper noun and common phenomena in the game industry referring to a [game](https://en.wikipedia.org/wiki/Game) designed for a primary purpose other than pure [entertainment](https://en.wikipedia.org/wiki/Entertainment).[](https://en.wikipedia.org/wiki/Serious_game#cite_note-1)